app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java

```diff
@@ -64,14 +65,35 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         AppUtil.setTaskDescription(activity, session.title, ContextCompat.getColor(activity, session.category.getVividColorResId()));
     }
 
+    // Receive all viewModel's properties changed.
+    private final Observable.OnPropertyChangedCallback onPropertyChanged = new Observable.OnPropertyChangedCallback() {
```

Comment:
That's right!
All instances and states should managed by Model.
I don't use navigate with parameter, Because it is model unmanaged object.
Thus Model will be singleton.

その通りです。
全てのインスタンスや状態は、モデルによって管理される必要があります。
私は、画面遷移時にパラメータを渡す方法も使わないです。なぜなら、そのパラメータは、「モデルに管理されないデータ」だからです。（「モデルに管理されているデータ」のまま渡せれば使ったかも知れませんが、少なくとも Android はそうではないので。データのIDだけ渡して再検索するのは場合によってはやります。）
なので、モデルは自然とシングルトンになります。